src/org/codecentral/type/objects/Letter.java

```diff
@@ -1,18 +1,50 @@
 package org.codecentral.type.objects;
 
+import java.awt.Color;
+import java.awt.Font;
 import java.awt.Graphics;
+import java.util.concurrent.ThreadLocalRandom;
 
 public class Letter extends GameObject {
 
+    /**
+     * The number of pixels to move upon each movement.
+     *
+     * @see #moveForward()
+     */
+    private int speed = 10;
+
     private char letter;
 
+    public Letter(int x, int y) {
+        this(generateRandomLetter(), x, y);
+    }
+
     public Letter(char letter, int x, int y) {
         super(x, y);
         this.letter = letter;
     }
 
     @Override
     public void onDraw(Graphics g) {
+        g.setFont(new Font("SansSerif", Font.BOLD, 10));
```

Comment:
Shouldn't we use Comic Sans?
Or have a random chance to have Comic Sans?

Reply:
It's a bit too comical. Maybe we have an option in the settings or something, but it's good for now.